components/brave_wallet/common/zcash_utils.cc

```diff
@@ -48,19 +49,13 @@ std::optional<uint64_t> ReadCompactSize(base::span<const uint8_t>& data) {
     value = type;
     data = data.subspan(1);
   } else if (type == 253 && data.size() >= 3) {
-    uint16_t val = 0;
-    base::ReadBigEndian(&data[1], &val);
-    value = val;
+    value = base::numerics::U16FromBigEndian(data.subspan(1).first<2u>());
     data = data.subspan(1 + 2);
   } else if (type <= 254 && data.size() >= 5) {
```

Comment:
is `subspan + first` some pattern from chrome?
why not just like `subspan(1, 2u)` everywhere?

Reply:
or subspan<1, 2u>() if we need compile time sized span

Reply:
Done.